### PR TITLE
add api lifecycle tags to admission.k8s.io

### PR DESCRIPTION
**Pull request by deads2k**
This one stands out as unusual because we don't serve it and we do allow the use of this beta API via webhook originating from apiservers.  I think we can and should still mark this API deprecated and removed (perhaps immediately) to ensure that we don't accidentally serve this API if something else is added to the group.

@kubernetes/sig-api-machinery-pr-reviews 
/kind cleanup

```release-note
NONE
```

### CODE REVIEW

**Comment by deads2k** [comment]
hopefully this fails verify

File: staging/src/k8s.io/api/admission/v1beta1/doc.go [hunk 1]
```diff
@@ -17,6 +17,7 @@ limitations under the License.
 // +k8s:deepcopy-gen=package
 // +k8s:protobuf-gen=package
 // +k8s:openapi-gen=false
+// +k8s:prerelease-lifecycle-gen=true
```
Comment (liggitt) [inline staging/src/k8s.io/api/admission/v1beta1/doc.go:20]: add replacement tag pointing to v1 AdmissionReview?

**Pull request merged**